src/params.ts

```diff
@@ -18,6 +18,46 @@ export const aragenAccounts = [
     privateKey:
       '0xce8e3bda3b44269c147747a373646393b1504bfcbb73fc9564f5d753d8116608',
     publicKey: '0x8401Eb5ff34cc943f096A32EF3d5113FEbE8D4Eb'
+  },
```

Comment:
Added a few accounts to print out when the start task runs. Also useful for tests that need more than two accounts.